content/en/docs/components/serving/kfserving.md

```diff
@@ -23,7 +23,7 @@ Our strong community contributions help KFServing to grow. We have a Technical S
 ## Install with Kubeflow
 
 KFServing works with Kubeflow 1.1. Kustomize installation files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
-See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs. 
+See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs.
```

Comment:
- For a11y + rephrase
```suggestion
Check the examples running KFServing on Istio/Dex in the [`kubeflow/kfserving`](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex) repository. For installation on major cloud providers with Kubeflow, follow their installation docs.
```

Would it be possible to find links to AWS etc KFServing installation guides and include them here? This can help new users 💯, I think

Reply:
The text above refers to follow the installation docs of the major cloud providers.
Examples should then be in the existing links like https://www.kubeflow.org/docs/aws/
I do not think we should additional links on the current page.